internal/ethapi/api.go

```diff
@@ -1625,6 +1626,25 @@ func newRPCTransaction(tx *types.Transaction, blockHash common.Hash, blockNumber
 		result.ChainID = (*hexutil.Big)(inner.ChainId)
 	case *types.ArbitrumUnsignedTx:
 		result.ChainID = (*hexutil.Big)(inner.ChainId)
+	case *types.ArbitrumSubtypedTx:
+		subtype := inner.TxSubtype()
+		result.Subtype = hexutil.Uint64(subtype)
+		switch subtype {
+		case types.ArbitrumTippingTxSubtype:
+			al := tx.AccessList()
+			result.Accesses = &al
+			result.ChainID = (*hexutil.Big)(tx.ChainId())
+			result.GasFeeCap = (*hexutil.Big)(tx.GasFeeCap())
+			result.GasTipCap = (*hexutil.Big)(tx.GasTipCap())
```

Comment:
don't you also need to set result.YParity?

Reply:
good catch! thanks